packages/react-native-room-kit/example/ios/RNExample/Info.plist

```diff
@@ -89,10 +89,11 @@
 	<key>UIStatusBarHidden</key>
 	<true/>
 	<key>UIStatusBarStyle</key>
-	<string></string>
+	<string>UIStatusBarStyleDarkContent</string>
```

Comment:
will this give us dark content on status bar? @ygit

Reply:
yes